.github/workflows/ci-dind.yml

```diff
@@ -1,4 +1,4 @@
-name: in-container
+name: CI-DinD
```

Comment:
we're not running "Docker-in-Docker", but mount the socket, hence "in-container" :) Maybe we can use "wormhole" or similar...

Reply:
Good point